### PR TITLE
feat: add label option to SearchSelectInput component

### PR DESCRIPTION
**Pull request by jenseo**

### CODE REVIEW

File: src/components/controls/selects/SearchSelect/SearchSelect.tsx [hunk 1]
```diff
@@ -201,9 +217,10 @@ const Wrapper = styled.div<{ isFullWidth?: boolean }>`
 
 const OptionsWrapper = styled.div`
   position: absolute;
-  top: ${({ theme }) => 6 * theme.spacer}px;
+  top: ${({ theme }) => `calc(100 % + ${theme.spacer * 2})px`};
```
Comment (filiptammergard) [inline src/components/controls/selects/SearchSelect/SearchSelect.tsx:220]:
```suggestion
  top: ${({ theme }) => `calc(100% + ${theme.spacer * 2})px`};
```

File: src/components/controls/selects/SearchSelect/SearchSelect.tsx [hunk 2]
```diff
@@ -201,9 +217,10 @@ const Wrapper = styled.div<{ isFullWidth?: boolean }>`
...
   right: 0;
   left: 0;
+  width: 100%;
```
Comment (filiptammergard) [inline src/components/controls/selects/SearchSelect/SearchSelect.tsx:223]: Either `right` and `left` or `width` should do it. I think it makes sense to use `right` and `left` for absolute positioned elements, so I suggest going for that!

**Review by filiptammergard — APPROVED** [review]
I noticed this happens when you pass a `message` with the new positioning of the options wrapper.

image

But not seeing any apparent solution for it now...

File: src/components/controls/selects/SearchSelect/SearchSelect.tsx (hunk 2 above)
Comment (jenseo) [reply src/components/controls/selects/SearchSelect/SearchSelect.tsx:223]: Sure!

**Comment by jenseo** [comment]
> I noticed this happens when you pass a `message` with the new positioning of the options wrapper.
> 
> image
> 
> But not seeing any apparent solution for it now...

Hmm, yeah, I'll take a thinking round on that one!

**Pull request merged**